apps/web/src/app/(authenticated)/insights/components/order-filters.tsx

```diff
@@ -163,24 +156,36 @@ export default function OrderFilters(): React.ReactNode {
   const handleAdPreviewChange = (e: ChangeEvent<HTMLInputElement>): void => {
     resetInsights();
     let newURL: string;
-    if (e.target.checked) {
-      newURL = addOrReplaceURLParams(pathname, searchParams, urlKeys.fetchPreviews, 'true');
-    } else {
-      newURL = addOrReplaceURLParams(pathname, searchParams, urlKeys.fetchPreviews);
-    }
+    if (e.target.checked) newURL = addOrReplaceURLParams(pathname, searchParams, urlKeys.fetchPreviews, 'true');
+    else newURL = addOrReplaceURLParams(pathname, searchParams, urlKeys.fetchPreviews);
```

Comment:
```suggestion
    const newURL = e.target.checked
      ? addOrReplaceURLParams(pathname, searchParams, urlKeys.fetchPreviews, 'true')
      : addOrReplaceURLParams(pathname, searchParams, urlKeys.fetchPreviews);
```
also in general we want to keep the brackets on if/else

you may even do:
```
    const newURL = addOrReplaceURLParams(
      pathname,
      searchParams,
      urlKeys.fetchPreviews,
      e.target.checked ? 'true' : undefined,
    );
```